scripts/release_notes.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/env bash
+
+if [ ! -n "$GH_USER" ] || [ ! -n "$GH_TOKEN" ]; then
+    echo "You must export GH_USER - your github user id"
+    echo "You must export GH_TOKEN - your github access token"
+    exit 1
+fi
+
+if [ ! -n "$1" ]; then
+    echo "Must provide arg"
+    exit 1
+fi
+
+GH_CURL="curl -u $GH_USER:$GH_TOKEN"
+GIST_API="https://api.github.com/gists"
+
+create_gist () {
```

Comment:
I would probably add a comment up here to suggest these are the methods or functions for the script:

```bash
#
# FUNCTIONS
#
```

If it looks weird leave it out, but definitely include the one for MAIN.

Reply:
I'm skipping this in favor of moving all of the pre-checks down into main for readability.